serverless/knative-client.adoc

```diff
@@ -0,0 +1,42 @@
+[id="knative-client"]
+= Using Knative Client
+include::modules/common-attributes.adoc[]
+:context: knative-client
+
+toc::[]
+
+Knative Client (`kn`) is the Knative command line interface (CLI). The CLI exposes commands for managing your applications, as well as lower level tools to interact with components of {product-title}. With `kn`, you can create applications and manage {product-title} projects from the terminal. 
+
+Knative client does not have its own log in mechanism. To log in to `kn` you must install the {product-title} CLI and use the `oc` login. Installation options for the CLI vary depending on your operating system.
```

Comment:
the `oc` login command is mentioned here and I'm wondering why we're also mentioning creating the kubeconfig via create-kubeconfig (it's the place where I've put my first comment). It should be sufficient to use `oc` login.

Reply:
Yup, `oc login` is sufficient.